persist-cli/build.gradle

```diff
@@ -20,7 +20,7 @@ plugins {
     id 'checkstyle'
 }
 
-description = 'Ballerina - Persist-Tools Java Native'
+description = 'Ballerina - Persist-CLI Java Native'
```

Comment:
Review dependencies for cli and cli-test modules. IMO some of them are unnecessary

Reply:
Removed unnecessary dependencies in [420b9c2](https://github.com/ballerina-platform/persist-tools/pull/30/commits/420b9c2617c8de5f06973d312894d3f35cb4bc73)